### PR TITLE
Update to specs 0.9.7 and markets 0.6.0

### PR DESCRIPTION
**Pull request by arajasek**

### CODE REVIEW

File: chain/stmgr/stmgr.go [hunk 1]
```diff
@@ -1120,3 +1123,12 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 
 	return csi.FilCirculating, nil
 }
+
+func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) runtime.NetworkVersion {
+	// TODO: get this number from the right place
```
Comment (arajasek) [inline chain/stmgr/stmgr.go:1128]: when the number gets added into ntwk-0.6.0

Comment (arajasek) [reply chain/stmgr/stmgr.go]: donesies

File: chain/stmgr/stmgr.go [hunk 2]
```diff
@@ -1120,3 +1123,11 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 
 	return csi.FilCirculating, nil
 }
+
+func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) runtime.NetworkVersion {
+	if build.UpgradeBreezeHeight == 0 || height <= build.UpgradeBreezeHeight {
```
Comment (ZenGround0) [inline chain/stmgr/stmgr.go:1128]: Cool name.  Is the next one going to be hurricane?

Comment (ZenGround0) [reply chain/stmgr/stmgr.go:1128]: Confirming this means that `UpgradeBreezeHeight` will still run protocol version 0.  `UpgradeBreezeHeight + 1` is version 1

**Review by magik6k — APPROVED** [review]

File: chain/stmgr/stmgr.go (hunk 2 above)
Comment (Kubuxu) [inline chain/stmgr/stmgr.go:1128]: I think 0 should mean, use the Version1, unless everything breaks.

**Review by ZenGround0 — APPROVED** [review]

File: chain/stmgr/stmgr.go (hunk 2 above)
Comment (arajasek) [reply chain/stmgr/stmgr.go]: Done, everything shouldn't break (the actual changes are trivial)

**Review by Kubuxu — APPROVED** [review]

**Pull request merged**